vmm/src/lib.rs

```diff
@@ -43,8 +51,16 @@ impl Display for Error {
 }
 
 pub fn start_vm_loop(config: VmConfig) -> Result<()> {
+    let exit_evt = EventFd::new(EFD_NONBLOCK).map_err(Error::EventFd)?;
+    let reset_evt = EventFd::new(EFD_NONBLOCK).map_err(Error::EventFd)?;
+
     loop {
-        let mut vm = Vm::new(config.clone()).map_err(Error::VmNew)?;
+        let mut vm = Vm::new(
+            config.clone(),
+            exit_evt.try_clone().unwrap(),
```

Comment:
As we're thinking about having multiple VMs in the same process we need to be really careful to avoid panics as that could take down other VMs.